build-config.js

```diff
@@ -0,0 +1,36 @@
+/* jshint browser: false, node: true, strict: true */
+'use strict';
+
+/**
+ * @see https://github.com/ckeditor/ckeditor5-dev-bundler-rollup/issues/37 for config options
```

Comment:
If we want to point someone to options documentation then it should be in the README of that package.